testing/unit_tests/t_utils.py

```diff
@@ -1,9 +1,11 @@
+import os
```

Comment:
Maybe use "Path" instead, see other comment at lines 63-64.

Reply:
Yup. Fixed together with the other.